packages/parameters/src/utils/env.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  getNumberFromEnv,
+  getStringFromEnv,
+} from '@aws-lambda-powertools/commons/utils/env';
+
+/**
+ * It returns the value of the POWERTOOLS_PARAMETERS_MAX_AGE environment variable.
+ *
+ * @returns {number|undefined}
+ */
+const getParametersMaxAge = (): number | undefined => {
+  try {
+    return getNumberFromEnv({
+      key: 'POWERTOOLS_PARAMETERS_MAX_AGE',
+    });
+  } catch (error) {
+    return undefined;
+  }
+};
+
+/**
+ * It returns the value of the POWERTOOLS_PARAMETERS_SSM_DECRYPT environment variable.
+ *
+ * @returns {string}
+ */
+const getSSMDecrypt = (): string => {
+  return getStringFromEnv({
```

Comment:
I would suggest replacing this `getStringFromEnv` with a call to `getBooleanFromEnv` with `extendedParsing` set to `true` & `defaultValue` to `false`.

This would change the return value to a `boolean` and we could simplify further the call in the `SSMProvider` class, since this is used in a `if/else` statement.

Also like discussed in the other comment above, I'd consider/suggest moving the `getBooleanFromEnv` directly in the `SSMProvider` so we can remove this file entirely.